contract/src/main.rs

```diff
@@ -932,6 +898,12 @@ pub extern "C" fn transfer() {
     )
     .unwrap_or_revert();
 
+    if NFTIdentifierMode::Hash == identifier_mode
+        && utils::should_break_up_owned_token_hashes(target_owner_key)
+    {
+        utils::break_up_owned_token_hashes(target_owner_key)
+    }
+
     let target_owner_item_key = utils::get_owned_tokens_dictionary_item_key(target_owner_key);
     let source_owner_item_key = utils::get_owned_tokens_dictionary_item_key(source_key);
```

Comment:
nitpick, but maybe be consistent on source then target or target then source in all places

Reply:
Renamed variables to get consistent